libsolidity/formal/SMTEncoder.cpp

```diff
@@ -1406,37 +1408,56 @@ void SMTEncoder::bitwiseOperation(BinaryOperation const& _op)
 	auto commonType = _op.annotation().commonType;
 	solAssert(commonType, "");
 
+	auto [bvSize, isSigned] = typeBvSizeAndSigndness(commonType);
+
+	auto bvLeft = smtutil::Expression::int2bv(expr(_op.leftExpression(), commonType), bvSize);
+	auto bvRight = smtutil::Expression::int2bv(expr(_op.rightExpression(), commonType), bvSize);
+
+	optional<smtutil::Expression> result;
+	if (_op.getOperator() == Token::BitAnd)
+		result = bvLeft & bvRight;
+	else if (_op.getOperator() == Token::BitOr)
+		result = bvLeft | bvRight;
+	else if (_op.getOperator() == Token::BitXor)
+		result = bvLeft ^ bvRight;
+	else
+		solAssert(false, "");
+
+	if (result)
```

Comment:
Since now we have `solAssert(false)` for an operator that would not initialize `result`, we could replace this `if` by `solAssert(result, "");`.